tests/conftest.py

```diff
@@ -1,36 +1,19 @@
-from enum import Enum
-
-from database_setup_tools.session_manager import SessionManager
 from database_setup_tools.setup import DatabaseSetup
 
-from tests.config import SQLITE_DATABASE_URI
+from tests.config import POSTGRESQL_DATABASE_URI
 from tests.integration.scenario.entities import model_metadata
 
 
-class Databases(Enum):
-    """ Enum of the databases """
-    # POSTGRESQL = "postgresql"
-    SQLITE = "sqlite"
-
-
-def build_setup(database: Databases) -> DatabaseSetup:
-    """ Build the setups for the databases """
-    database_uri = {
-        # Databases.POSTGRESQL: POSTGRESQL_DATABASE_URI,
-        Databases.SQLITE: SQLITE_DATABASE_URI
-    }.get(database)
-
-    return DatabaseSetup(model_metadata=model_metadata, database_uri=database_uri)
-
-
-setups = [build_setup(database=database) for database in Databases]
-session_managers = [SessionManager(database_uri=setup.database_uri) for setup in setups]
+class RepositoryTest:
+    pass
 
 
 # pylint: disable=unused-argument
 # noinspection PyUnusedLocal
 def pytest_sessionstart(session):
-    """ Create or reset the databases before the tests """
-    for setup in setups:
-        setup.drop_database()
-        setup.create_database()
+    """Create or reset the databases before the tests"""
+    setup = DatabaseSetup(
+        model_metadata=model_metadata, database_uri=POSTGRESQL_DATABASE_URI
+    )  # TODO: This is fucked up. The DB Setup fails once the db exists because it creates it by default.
```

Comment:
I think this was one of the fuckups we had because we didn't have integration tests 🙃 this should be fixed with the new release